scripts/prepare_heat_demand.py

```diff
@@ -75,24 +83,37 @@ def get_years(file_list):
     """
     years_array = np.arange(1990, 2051)
     years_list = []
+    newline = "\n"
```

Comment:
I've explained in a comment and more descriptive names with commit https://github.com/rl-institut/oemof-B3/pull/76/commits/a22b2ed677dd336b9f9c741508ef149a357fae1c for now. In future we could improve the search for a year in a string in this function.

Reply:
I meant the `newline`. I remember that you came up with this because had a problem with the linters. Maybe you even don't need a line break in an error message? Or, if you do, find another way to satisfy the linters?

Reply:
Done with commit https://github.com/rl-institut/oemof-B3/pull/76/commits/da7ab86826b5a4ead1091d96e76d92810498b58b.